tests/test_explorer.py

```diff
@@ -0,0 +1,172 @@
+"""Unit tests for explorer module."""
+
+import unittest
+from unittest import mock
+
+from PyQt5.QtCore import QObject, Qt
+from PyQt5.QtWidgets import QApplication, QTreeWidgetItem
+from PyQt5.QtTest import QTest
+
+from iquip.apps import explorer
+
+class ExplorerFrameTest(unittest.TestCase):
+    """Unit tests for ExplorerFrame class."""
+
+    def setUp(self):
+        self.qapp = QApplication([])
+        patcher = mock.patch("iquip.apps.explorer._FileFinderThread")
+        self.MockedFileFinderThread = patcher.start()
+        self.addCleanup(patcher.stop)
+
+    def tearDown(self):
+        del self.qapp
+
+    @mock.patch("iquip.apps.explorer.ExplorerApp.lazyLoadFile")
+    def test_file_tree_item_expanded(self, mockedLazyLoadFile):
```

Comment:
We have to decide where to put these _functional tests_.

Reply:
I looked for several solutions:
1. Use `unittest` similar to the current tests.
Even if the module name is __unittest__, but it can also tests functional tests.
2. Use [pytest](https://docs.pytest.org/en/7.3.x/)
It is one of the most powerful test packages. Probably it offers more test methods.
But I don't like using various kinds of packages...

Then, How about using `unittest` and making functional tests in another file such as `test_explorer_function.py`?

Reply:
Or I think implementing functional tests in this file is not bad..